spec/stat_tracker_spec.rb

```diff
@@ -32,6 +32,8 @@
          location = './data/teams_fixture.csv'
 
          expect(@stat_tracker.read_teams_csv(location)[0]).to be_a Team
+         expect(@stat_tracker.read_teams_csv(location)[0].team_id).to eq(1)
```

Comment:
I wonder if it's best practice to test this is a different `it` block, since we are looking for something different here. (it 'has an id' do, maybe)